[
bidi_upgrade
.
py
]
expected
:
ERROR
